### PR TITLE
REF: define reductions non-dynamically

### PR DESCRIPTION
**Pull request by jbrockmendel**
- [ ] closes #xxxx (Replace xxxx with the GitHub issue number)
- [ ] [Tests added and passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#writing-tests) if fixing a bug or adding a new feature
- [ ] All [code checks passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#pre-commit).
- [ ] Added [type annotations](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#type-hints) to new arguments/methods/functions.
- [ ] Added an entry in the latest `doc/source/whatsnew/vX.X.X.rst` file if fixing a bug or adding a new feature.

### CODE REVIEW

File: pandas/core/series.py [hunk 1]
```diff
@@ -6091,5 +6091,138 @@ def all(
...
+    @doc(make_doc("sum", ndim=1))
+    def sum(
+        self,
+        axis: Axis | None = None,
+        skipna: bool = True,
+        numeric_only: bool = False,
+        min_count: int = 0,
+        **kwargs,
+    ):
+        return NDFrame.sum(self, axis, skipna, numeric_only, min_count, **kwargs)
```
Comment (topper-123) [inline pandas/core/series.py:6125]: `return super().sum(axis, skipna, numeric_only, min_count, **kwargs)`?

Same question below.

File: pandas/core/series.py [hunk 2]
```diff
@@ -6091,5 +6091,138 @@ def all(
...
+    @doc(make_doc("min", ndim=1))
+    # error: Signature of "min" incompatible with supertype "IndexOpsMixin"
+    def min(  # type: ignore[override]
+        self,
+        axis: Axis | None = 0,
+        skipna: bool = True,
+        numeric_only: bool = False,
+        **kwargs,
+    ):
+        return NDFrame.min(self, axis, skipna, numeric_only, **kwargs)
```
[the remark this answers is not in the document]
Comment (jbrockmendel) [reply pandas/core/series.py:6103]: yep, ive got a branch doing exactly that

[the remark this answers is not in the document]
Comment (topper-123) [reply pandas/core/series.py:6103]: 👍

**Comment by topper-123** [comment]
Thanks @jbrockmendel

**Pull request merged**